resources/gui/changehotkey.rml

```diff
@@ -0,0 +1,162 @@
+<rml>
+    <head>
+        <style>
+            body
+            {
+                font-family: FreeMono;
+                font-size: 24;
+                text-align: center;
+                background-color: rgba(0,0,0,255);
```

Comment:
Would be better to still show the game in the background instead of blacking out the screen like this.
Could maybe make a black box in the middle instead of the whole screen?